coins.json

```
@@ -1206,6 +1206,32 @@
       "clientDocs": "https://docs.harmony.one/home/harmony-networks/harmony-network-overview/mainnet"
     }
   },
+  {
+    "id": "Oasis",
+    "name": "Oasis",
+    "coinId": 5786211,
+    "slip44": 474,
```

if `slip44` is not duplicated with other coins, you can do this: 
```suggestion
    "coinId": 474,
```